.github/workflows/virusTotalSubmit.js

```diff
@@ -0,0 +1,105 @@
+const glob = require('glob');
+const { v4: uuidv4 } = require('uuid');
+const fs = require('fs');
+const { exec } = require('child_process');
+
+
+function sleep(n) {
```

Comment:
Can this get a more descriptive name?
```suggestion
function sleep(timeout) {
```